lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -27,7 +27,13 @@ def change
     create_table :oauth_access_tokens do |t|
       t.integer  :resource_owner_id
       t.integer  :application_id
+
+      # In the event you would like to use a custom generator as detailed in
+      # http://git.io/vqDqe, you may need to increase the size of this column
+      # to allow for tokens larger than 255 characters.
       t.string   :token,             null: false
+      #t.text     :token,             null: false
```

Comment:
Let's do something like:

```diff
diff --git a/lib/generators/doorkeeper/templates/migration.rb b/lib/generators/doorkeeper/templates/migration.rb
index af5aa7d..d5a51c4 100644
--- a/lib/generators/doorkeeper/templates/migration.rb
+++ b/lib/generators/doorkeeper/templates/migration.rb
@@ -27,7 +27,13 @@ class CreateDoorkeeperTables < ActiveRecord::Migration
     create_table :oauth_access_tokens do |t|
       t.integer  :resource_owner_id
       t.integer  :application_id
+
+      # If you use a custom token generator you may need to change this column
+      # from string to text, so that it accepts tokens larger than 255
+      # characters. More info on custom token generators in:
+      # https://github.com/doorkeeper-gem/doorkeeper/tree/v3.0.0.rc1#custom-access-token-generator
+      #
+      # t.text     :token,             null: false
       t.string   :token,             null: false
+
       t.string   :refresh_token
       t.integer  :expires_in
       t.datetime :revoked_at
```

Thanks again!